src/Components/ProjectForm.js

```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import { ReactSession } from "react-client-session";
 import styled from "styled-components";
 import TextField from "@mui/material/TextField";
+import Button from '@mui/material/Button';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8488141)
```suggestion
import Button from "@mui/material/Button";
```